fastlane/metadata/de-DE/subtitle.txt

```diff
@@ -1 +1 @@
-Erstelle eine Website oder ein Blog
\ No newline at end of file
+Erstelle deine neue Website
```

Comment:
The new value is the correct one. I updated the translation in GlotPress itself to avoid having to dance between > 30 chars and <= 30 chars manually on every release. See discussion [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/17080#discussion_r694670803)